Automattic/wp-calypso
# Subscriptions: Add content summary modal to Paid Subscribers importer

Related to https://github.com/Automattic/wp-calypso/issues/93150

## Proposed Changes

It shows the modal with the upload summary on content upload.

<img width="813" alt="Screenshot 2024-08-21 at 14 19 46" src="https://github.com/user-attachments/assets/20838d08-aae5-4b9a-8c3c-6a1f7711558d">
<br><br>

Required backend change D158995-code

## Why are these changes being made?

Paid Subscribers importer iteration.

## Testing Instructions

1. Go to `/import/newsletter/substack/{site}`
2. Insert the Substack URL
3. Upload file exported from Substack
4. Notice the modal opened
5. Cliock continue making sure the authors mapping is shown

## Pre-merge Checklist
